src/charts/scale.js

```diff
@@ -61,7 +61,7 @@ export default function Scale () {
       domain = config.xDomain
     }
 
-    const markWidthOffset = config.markWidth ? config.markWidth / 2 : 0
```

Comment:
That's where the x axis outer padding is set only on bars

Reply:
just a preference but I like using [`Array.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) for this type of operation.